src/v2/Apps/Auction/AuctionApp.tsx

```diff
@@ -113,9 +113,7 @@ export const AuctionAppFragmentContainer = createFragmentContainer(AuctionApp, {
 
       internalID
       coverImage {
-        cropped(width: 1800, height: 600, version: "normalized") {
-          src
-        }
+        url(version: ["larger", "large"])
```

Comment:
So I checked Gemini to see what versions we actually have:

```
gemini:staging> Template.where(key: 'sale').first.versions.map { |x| [x.key, x.width, x.height] }
=> [["large_rectangle", 640, 480], ["wide", 2000, nil], ["square", 230, 230], ["source", 1920, 1920]]
```

So looks like you want this to be `version: ["wide", "source", "large_rectangle"]`

Reply:
(Also it's worth noting that none of these are really adequate; we'd probably want to add a normalized to this.)

Reply:
Updated 👍